### PR TITLE
IDTCTJNKNS-257: Update to detect8 script url

### PR DESCRIPTION
**Pull request by DanaMaxfield**
Update plugin to the Detect 8 url

**The tests can not run and this PR can not be merged until the Detect 8 script is available**

### CODE REVIEW

**Review by stevebillings — APPROVED** [review]

File: src/main/java/com/synopsys/integration/jenkins/detect/service/strategy/DetectScriptStrategy.java [hunk 1]
```diff
@@ -35,10 +35,10 @@
...
 public class DetectScriptStrategy extends DetectExecutionStrategy {
     public static final String DETECT_INSTALL_DIRECTORY = "Detect_Installation";
-    public static final String SUPPORTED_SHELL_SCRIPT_URL = "https://detect.synopsys.com/detect7.sh";
-    public static final String SHELL_SCRIPT_FILENAME = "detect7.sh";
-    public static final String SUPPORTED_POWERSHELL_SCRIPT_URL = "https://detect.synopsys.com/detect7.ps1";
-    public static final String POWERSHELL_SCRIPT_FILENAME = "detect7.ps1";
+    public static final String SUPPORTED_SHELL_SCRIPT_URL = "https://detect.synopsys.com/detect8.sh";
+    public static final String SHELL_SCRIPT_FILENAME = "detect8.sh";
+    public static final String SUPPORTED_POWERSHELL_SCRIPT_URL = "https://detect.synopsys.com/detect8.ps1";
+    public static final String POWERSHELL_SCRIPT_FILENAME = "detect8.ps1";
```
Comment (stevebillings) [inline src/main/java/com/synopsys/integration/jenkins/detect/service/strategy/DetectScriptStrategy.java:42]: Certainly not a big deal, but maybe next time someone works on this we could move to a single int representing the Detect major version that gets used in 3 places (feels like someone could overlook one of the three places if not careful).

Comment (stevebillings) [reply src/main/java/com/synopsys/integration/jenkins/detect/service/strategy/DetectScriptStrategy.java:42]: (Sorry, just now realized that this PR has been waiting for my review for 6 weeks... I'm a bit late with this feedback!)

**Comment by Dohbedoh** [comment]
@DanaMaxfield @stevebillings @dmamidisynopsys The plugin has not been released publicly for a long time. Any chance to get a release ?

**Review by jppetrakis — COMMENTED** [review]
Dont know how I missed this one....

**Pull request merged**